### PR TITLE
InnerSource patterns book translation into Brazilian Portuguese

### PR DESCRIPTION
**Pull request by jrcosta**
# Pull Request: InnerSource patterns book translation into Brazilian Portuguese

Hello!

We'd like to submit our translation of the "InnerSource Patterns Book" for review and inclusion in the repository. This pull request includes the following files and changes:

 - [x] Translated all the structured patterns into Brazilian Portuguese. The translation can be found under /translation/pt
 - [x] Proofread all the document
 - [x] Linted markdown files
 - [x] Edited the ToC generation script and templates

## Motivation:
We made this translation with the goal of making the valuable content of the "InnerSource Patterns Book" accessible to the Portuguese-speaking community. I believe that this translation will significantly benefit those who wish to learn about InnerSource and its practices.

## Progress:
The translation has undergone initial completion and is now ready for the next phase of thorough review and revisions. While the primary content has been translated, there might still be areas that require fine-tuning for accuracy, coherence, and consistency. Your valuable feedback and suggestions will greatly contribute to refining the quality of the translated content. I am committed to working closely with the community to address any issues that arise during the review process.

Thank you for your time and consideration in reviewing this pull request.

Best regards,
Eneri Junior

### CODE REVIEW

**Comment by spier** [comment]
Wow, what an amazing PR @jrcosta !
Will take a look shortly and leave feedback.

**Comment by jrcosta** [comment]
Update comment**

@spier I've thought about creating something to simplify the review process. Since these files aren't modified but rather new, GitHub doesn't show their diff. I've created a script that generates a diff file, displaying how it was in EN and how it ended up in PT-BR, like the example below:

https://github.com/jrcosta/InnerSourcePatterns/blob/book-ptbr-teste-diff/diff-translate-files/diff-30-day-warranty.md.diff

What do you think?

**Comment by spier** [comment]
Hi again @jrcosta. I was itching to see this live somewhere :) Therefore I made the required changes, so that I could publish it to our Staging account. You can now see it here:
https://innersourcecommons.gitbook.io/innersource-patterns-staging/v/brazilian-portuguese/

I have not checked if everything works but generally it does look ok to me 🥳 

As mentioned, this is not the Production version of our book. The content at the URL above will not be picked up by Google either. So this is only for testing purposes.

Note that I had to create a copy of your `book-ptbr` branch in this repo in order to publish this to Staging. That also means however that any changes that you push to your branch will NOT automatically be shown in Staging.

Next:
I did see your question about creating a diff to simplify the review process. Let me get back to you on that.

Do you already have somebody (or multiple people) that could do the review?

**Comment by spier** [comment]
Quick mental notes for myself. Will spell these out tomorrow:
- see [link check output](https://github.com/InnerSourceCommons/InnerSourcePatterns/actions/runs/5944542800)
   - translate RFC template or link back to English version?
   - some issue in [translation/pt/README.md](https://github.com/jrcosta/InnerSourcePatterns/blob/book-ptbr/translation/pt/README.md) - we could opt to ignore this? (btw is this how @yuhattor had intended this README to be used? i.e. as a translation of `translation/README.md`?)
- branch name `book-ptbr` works but only by chance :) would `book-pt` be fine too?

**Comment by jrcosta** [comment]
> 
Hello @spier, we still don't have anyone available for the review. However, now that the PR has been open, I suggest that we inform the team on Brazilian channel Slack to gather people for the review process.

I updated the comment about diff files, because the viewing experience is improved when the file extension is ".diff".

**Comment by yuhattor** [comment]
Amazing PR!!!! Thank you so much for the contribution.
I think branch name should be book-ptbr in my opinion. 

If, hypothetically (maybe not), someone comes along who wants to provide Portuguese that is not a Brazilian expression, then they need to use book-pt.
The one I'm doing for innersourcecommons.org is pt-br.
But having said that, the correct branch name for English should be en-us, and Japanese should be ja-jp.

The problem may be what Portuguese speakers who are not Brazilian Portuguese will think. In the case of English, I think that in many cases, people can read either British English or American English without much concern, but in the case of Portuguese, I wonder how it would be.

I'd be interested to hear native speakers' opinions.

**Comment by spier** [comment]
@yuhattor in [translation/README.md](https://github.com/jrcosta/InnerSourcePatterns/blob/main/translation/README.md) you mention that each translation project should provide a `/translation/<YOUR_LANGUAGE>/README.md` of their own.

However I noticed that for `jp` and `zn` we have not created such a file.

Now for `pt`, this file exists and it contains a translation of translation/README.md.
Is that how you had intended it to be used?

**Comment by yuhattor** [comment]
Thanks.

To be honest, I can't remember much about it at the time, maybe it's just something I was working on...
I guess this is a little different from my assumptions that I thought of at the time, but still, writing a README.md for each language contributor and putting a message or acknowledgement in it might be a good option!

**Comment by spier** [comment]
hi @jrcosta.

Again, what an amazing PR. After a handful of small fixes I already had your translations running on our Staging site: 
https://innersourcecommons.gitbook.io/innersource-patterns-staging/v/brazilian-portuguese/

So from a technical perspective this PR is good to be merged.

## Translation review

As I don't speak Brazilian Portuguese, I cannot judge the quality of the translation.

I saw your comment above about how you might go about the review process. As I have not done translations before, I was wondering if @yuhattor has any input on this?

Also would we offer the reviewers to be acknowledged for their time/work at the bottom of the file, just like [what you did](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/577/files#diff-ea6164e0d47fe0e4975498307874dcebb31f9959869862b1d2b1e7d5bd7afb21R73-R76) for yourself and Humberto already?

## Further notes

These are observations that I made while reviewing your PR. None of these would block this PR from being merged though!

- in the meantime 3 new patterns were added to the English version
  - I would recommend to translate them in a separate PR, so that we can get this PR here over the finish line quicker. But up to you. 
  - new patterns are: [Group Support](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/group-support.md), [Extensions for Sustainable Growth](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/extensions-for-sustainable-growth.md), [Standard Release Process](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/release-process.md)
- there was one broken link, pointing to `templates/rfc.md` (see [link check output](https://github.com/InnerSourceCommons/InnerSourcePatterns/actions/runs/5944542800)).
   - I fixed this by linking to the English RFC template instead (see 8c7bbd9) . Btw the Japanese translation used the same approach.
   - However if you want to translate the RFC template as well, feel free
- while technically we have used the 2-letter language codes so far in the branch names as well, the branch name `book-ptbr` works as well (only by chance actually but hey :)). let's leave it as is (also see further [comment](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/577#issuecomment-1689232250) by @yuhattor)
- I noticed that on you repo you ran a GHA called [pages-build-deployment](https://github.com/jrcosta/InnerSourcePatterns/actions/workflows/pages/pages-build-deployment). Do you know where this came from? did you try to render GitHub Pages on your repo/fork? 


## Release notes

- when merging this PR, let's make sure that all contributors are acknowledged as `Co-authored-by`, so that they get the well deserved GitHub fame :)

**Comment by jrcosta** [comment]
> Amazing PR!!!! Thank you so much for the contribution. I think branch name should be book-ptbr in my opinion.
> 
> If, hypothetically (maybe not), someone comes along who wants to provide Portuguese that is not a Brazilian expression, then they need to use book-pt. The one I'm doing for innersourcecommons.org is pt-br. But having said that, the correct branch name for English should be en-us, and Japanese should be ja-jp.
> 
> The problem may be what Portuguese speakers who are not Brazilian Portuguese will think. In the case of English, I think that in many cases, people can read either British English or American English without much concern, but in the case of Portuguese, I wonder how it would be.
> 
> I'd be interested to hear native speakers' opinions.

@yuhattor Thank, you!!

While the translation was underway, I came to the realization that I should adhere to the ISO 639-1 standard, which requires the use of 2-letter language codes. As a result, I modified the folder name to 'pt'. However, I unintentionally neglected to update the branch name. In my view, this shouldn't pose an issue, as upon a cursory review, a Brazilian reader would readily recognize 'pt' as 'pt-br'.

**Comment by jrcosta** [comment]
> Also would we offer the reviewers to be acknowledged for their time/work at the bottom of the file, just like [what you did](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/577/files#diff-ea6164e0d47fe0e4975498307874dcebb31f9959869862b1d2b1e7d5bd7afb21R73-R76) for yourself and Humberto already?
>
Hi @spier on this point, do you say to make acknowledged the reviewers who are in this PR, or those who will review the quality of the translation?

> * in the meantime 3 new patterns were added to the English version
>   
>   * I would recommend to translate them in a separate PR, so that we can get this PR here over the finish line quicker. But up to you.
>   * new patterns are: [Group Support](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/group-support.md), [Extensions for Sustainable Growth](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/extensions-for-sustainable-growth.md), [Standard Release Process](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/release-process.md)

About that, I will translate and bring in a new PR.

>   * I fixed this by linking to the English RFC template instead (see [8c7bbd9](https://github.com/InnerSourceCommons/InnerSourcePatterns/commit/8c7bbd94986d958823cb53cd1a73e128dc4d6c79)) . Btw the Japanese translation used the same approach.

I believe we can keep this one untranslated.

> * I noticed that on you repo you ran a GHA called [pages-build-deployment](https://github.com/jrcosta/InnerSourcePatterns/actions/workflows/pages/pages-build-deployment). Do you know where this came from? did you try to render GitHub Pages on your repo/fork?

This was just a test we ran, but I'll be removing this from the branch.

> ## Release notes
> * when merging this PR, let's make sure that all contributors are acknowledged as `Co-authored-by`, so that they get the well deserved GitHub fame :)

In this point, can I adjust now? Or could this just have been done before submitting the pr? I ask because I've never done this.

**Comment by jrcosta** [comment]
> I saw your comment above about how you might go about the review process. As I have not done translations before, I was wondering if @yuhattor has any input on this?


On this point, when I talked about making the .diff available, it was so that language reviewers could have this view of the file:



I made available in this folder all the .diff files of everything that was translated in this PR, in case it helps the language reviewers:

https://github.com/jrcosta/InnerSourcePatterns/tree/book-ptbr-teste-diff/diff-translate-files

**Comment by spier** [comment]
> > Also would we offer the reviewers to be acknowledged for their time/work at the bottom of the file, just like [what you did](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/577/files#diff-ea6164e0d47fe0e4975498307874dcebb31f9959869862b1d2b1e7d5bd7afb21R73-R76) for yourself and Humberto already?
> 
> Hi @spier on this point, do you say to make acknowledged the reviewers who are in this PR, or those who will review the quality of the translation?

Both should be acknowledged.
Here I specifically meant the new reviewers that you are looking for now, that should review the quality of the translation.

> 
> > * in the meantime 3 new patterns were added to the English version
> >   
> >   * I would recommend to translate them in a separate PR, so that we can get this PR here over the finish line quicker. But up to you.
> >   * new patterns are: [Group Support](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/group-support.md), [Extensions for Sustainable Growth](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/extensions-for-sustainable-growth.md), [Standard Release Process](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/release-process.md)
> 
> About that, I will translate and bring in a new PR.

👍 

> 
> > * I fixed this by linking to the English RFC template instead (see [8c7bbd9](https://github.com/InnerSourceCommons/InnerSourcePatterns/commit/8c7bbd94986d958823cb53cd1a73e128dc4d6c79)) . Btw the Japanese translation used the same approach.
> 
> I believe we can keep this one untranslated.

👍 

> 
> > * I noticed that on you repo you ran a GHA called [pages-build-deployment](https://github.com/jrcosta/InnerSourcePatterns/actions/workflows/pages/pages-build-deployment). Do you know where this came from? did you try to render GitHub Pages on your repo/fork?
> 
> This was just a test we ran, but I'll be removing this from the branch.

👍 

> 
> > ## Release notes
> > 
> > * when merging this PR, let's make sure that all contributors are acknowledged as `Co-authored-by`, so that they get the well deserved GitHub fame :)
> 
> In this point, can I adjust now? Or could this just have been done before submitting the pr? I ask because I've never done this.

We can do this at the very end. 

Once we are happy with the state of this PR, I will use "squash and merge".
At that point I can list all contributors, and they will be listed in GitHub according.

That merge commit would then look similar to this:
https://github.com/InnerSourceCommons/InnerSourcePatterns/commit/0ebc0bdfd64ee410b83c48ebb9fcb4c615be9d37

The only thing we need for this are the GitHub usernames of all contributors that should be included.

So I guess `@jrcosta` and `@zilio`?

**Comment by spier** [comment]
@jrcosta Would it be ok to add you to our [CODEOWNERS](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/.github/CODEOWNERS#L13-L15) file?

That would mean that if new PRs are sent in the future that contain changes to the Brazilian Portuguese files, you would automatically be added to the PR as a reviewer. That would help us a lot, as the core maintainers of the repo don't speak that language :)

We can add multiple people per language, so if @zilio would like to do future reviews, I am happy to add you too :)

File: translation/pt/patterns/core-team.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Cada um desses itens é muito importante para um ecossistema de produto saudável, mas é improvável que seja priorizado como uma contribuição.
+
+A equipe central pode ser composta por um pequeno número de pessoas em tempo integral ou parcial.
+A escolha depende da quantidade de trabalho necessário, da disponibilidade de recursos e da cultura da organização.
+A consideração mais importante é formar a equipe de forma que permita que a organização a capacite e a responsabilize da mesma maneira que qualquer outra equipe.
+
+Due to their central role, core team members should nearly always fill the role of **Trusted Committers** as well (for more on that concept see [Learning Path][tc-learning-path] and [Pattern][tc-pattern]).
+While the Trusted Committer role focuses mostly on facilitating others' contribution and use of the project, a core team member regularly contributes to the project as well.
+The core team doesn't have its own business agenda that determines its contributions.
+They decide what to work on based on what will help others most to use and contribute to the project.
```
Comment (spier) [inline translation/pt/patterns/core-team.md]: @jrcosta this block here is still in English. Can this be deleted?

File: translation/pt/patterns/repository-activity-score.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Uma lista de projetos InnerSource não deve ser considerada um recurso estático, mas sim um local emocionante para descobrir e explorar projetos novos e ativos, assim como uma página de notícias que lista os tópicos mais interessantes do dia primeiro. Portanto, é benéfico quando a ordem dos projetos é atualizada regularmente e muda de acordo com a popularidade e atividade do projeto.
+
+Essas considerações levaram a um primeiro protótipo para calcular uma pontuação de atividade do repositório, que funcionou surpreendentemente bem e determina uma ordem sempre em mudança de projetos de acordo com a sua atividade.
+
+## Contexto
+
+A descoberta de projetos InnerSource pode ser facilitada com o [Portal InnerSource](innersource-portal.md) e o padrão [Gig Marketplace](gig-marketplace.md), ou promovendo projetos em outros canais de comunicação e plataformas. A pontuação de atividade define uma ordem padrão na qual os projetos são apresentados à comunidade.
+
+## Forces
```
Comment (spier) [inline translation/pt/patterns/repository-activity-score.md]:
```suggestion
## Forças
```

Comment (spier) [reply translation/pt/patterns/repository-activity-score.md]: Seems like a straightforward fix, as I looked up the word in other patters.
Will commit this :)

File: translation/pt/patterns/core-team.md (hunk 1 above)
Comment (zilio) [reply translation/pt/patterns/core-team.md]: That's right @spier , we had forgotten to remove this already translated section. I just removed it."

**Comment by zilio** [comment]
> @jrcosta Would it be ok to add you to our [CODEOWNERS](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/.github/CODEOWNERS#L13-L15) file?
> 
> That would mean that if new PRs are sent in the future that contain changes to the Brazilian Portuguese files, you would automatically be added to the PR as a reviewer. That would help us a lot, as the core maintainers of the repo don't speak that language :)
> 
> We can add multiple people per language, so if @zilio would like to do future reviews, I am happy to add you too :)

I would be happy to help maintain the Brazilian Portuguese repository.

**Comment by fer-correa** [comment]
Hi @jrcosta  and @zilio,
First of all... Congrats 😃 

I have a small doubt and I confess that it is more focused on maintaining a possible standard of translations for pt-br.
I could really be wrong, but I think this incredible work you did is in the pt directory (which would be for Portuguese in Portugal) and not in pt-br 🇧🇷 .
@yuhattor , am I correct about it?

If I'm wrong, sorry and congratulations again to you.

**Comment by yuhattor** [comment]
Thank you all
I summarized Pros and Cons here...

If there are no psychological blockers, adopting # 2 is the best from an SEO perspective, and this seems to be the best for the recent and future.
In the future, however, we should aim for # 3. It feels more inclusive that way.
At least zh-CN and ja-JP don't have tremendous traffic or a lot of external links. en is the default language, and since the locale doesn't appear in the URL, it's not much of a problem.
I wonder if other languages will be moved to ja-JP, cn-CN as soon as the SEO problem is solved.

Would like to hear your opinion.

### Option 1: Use `pt` for Brazilian Portuguese
| Pros                           | Cons                                  |
|--------------------------------|---------------------------------------|
| Simplifies the code structure  | Doesn't distinguish between dialects   |
| Consistent with existing codes | May confuse or alienate non-Brazilian Portuguese speakers |
| Less verbose                   | Makes it harder to add European Portuguese in the future |

### Option 2: Use `pt-br` for Brazilian Portuguese
| Pros                                   | Cons                             |
|----------------------------------------|----------------------------------|
| Clearly indicates the dialect          | Inconsistent with existing codes |
| Allows for future dialects (`pt-pt`)  | More verbose                      |
| Targets a specific audience precisely  | While English also has en-US and en-GB, en is adopted for American English, It seems a little strange that only Brazilian Portuguese includes pt-br while pt is absent |

### Option 3: Change All Codes to Include Region (e.g., `ja-JP`, `zh-CN`)
| Pros                                     | Cons                              |
|------------------------------------------|-----------------------------------|
| Standardizes the format for all languages | It would break all previous links |
| Explicitly identifies dialects            | Makes URLs and directory structure more verbose |
| Allows for flexibility in adding new dialects | Requires change codes and documentation |

GitBook.com appears to allow content redirection but not URL redirection. This means there is no way to redirect /en -> /en-US. So existing content is affected and some hack is needed.

**Comment by jrcosta** [comment]
> @jrcosta Would it be ok to add you to our [CODEOWNERS](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/.github/CODEOWNERS#L13-L15) file?
> 
> That would mean that if new PRs are sent in the future that contain changes to the Brazilian Portuguese files, you would automatically be added to the PR as a reviewer. That would help us a lot, as the core maintainers of the repo don't speak that language :)
> 
> We can add multiple people per language, so if @zilio would like to do future reviews, I am happy to add you too :)

Sure, you can add me. I would be happy to help! 🚀 😄

**Comment by jrcosta** [comment]
> Thank you all I summarized Pros and Cons here...
> 
> If there are no psychological blockers, adopting # 2 is the best from an SEO perspective, and this seems to be the best for the recent and future. In the future, however, we should aim for # 3. It feels more inclusive that way. At least zh-CN and ja-JP don't have tremendous traffic or a lot of external links. en is the default language, and since the locale doesn't appear in the URL, it's not much of a problem. I wonder if other languages will be moved to ja-JP, cn-CN as soon as the SEO problem is solved.
> 
> Would like to hear your opinion.
> 
> ### Option 1: Use `pt` for Brazilian Portuguese
> Pros	Cons
> Simplifies the code structure	Doesn't distinguish between dialects
> Consistent with existing codes	May confuse or alienate non-Brazilian Portuguese speakers
> Less verbose	Makes it harder to add European Portuguese in the future
> ### Option 2: Use `pt-br` for Brazilian Portuguese
> Pros	Cons
> Clearly indicates the dialect	Inconsistent with existing codes
> Allows for future dialects (`pt-pt`)	More verbose
> Targets a specific audience precisely	While English also has en-US and en-GB, en is adopted for American English, It seems a little strange that only Brazilian Portuguese includes pt-br while pt is absent
> ### Option 3: Change All Codes to Include Region (e.g., `ja-JP`, `zh-CN`)
> Pros	Cons
> Standardizes the format for all languages	It would break all previous links
> Explicitly identifies dialects	Makes URLs and directory structure more verbose
> Allows for flexibility in adding new dialects	Requires change codes and documentation
> GitBook.com appears to allow content redirection but not URL redirection. This means there is no way to redirect /en -> /en-US. So existing content is affected and some hack is needed.

hi @yuhattor, Monday will talk with @zilio about that, for reply better here, ok?

**Comment by spier** [comment]
Love the level of consideration that everybody is given the topic of `pt` vs `pt-br`.
Just want to make sure that we are not overthinking this and ending in analysis paralysis :)

My opinion on this topic starts with a question (sorry if that has already been answered):
Is Brazilian Portuguese so different from European Portuguese that speakers of both languages won't understand written text in the respective other language?

## My opinion about implementation

I am leaning for option (2), as it is quick to implement and "correct enough for my point of view.

Changes required on a folder/configuration level:

* renaming folder to `pt-br`
* changing configuration values in the scripts from `pt` to `pt-br`

To add another angle, let's consider the user as well (i.e. the reader of our book).
The user sees the language in these places[^1]:

* language selection at the top left
* sub-string as part of the URL
* (new) possible a country flag in front of the language selection

I am suggesting to implement this as shown in the screenshot below. 
Note that to make it even more clear which language we are talking about, I have a country flag in front of the language selection. May not work so well for languages spoken in many countries but I guess for now it is ok?

I have already configured it like this on Staging, so that you can click around:
https://innersourcecommons.gitbook.io/innersource-patterns-staging/v/pt-br/

(once deployed to production the URL will be `https://patterns.innersourcecommons.org/v/pt-br/`)

pt-br impl proposal 2023-08-27

Note to self:
The flags (emojis) that I copied into the space titles are from here: https://emojiguide.com/flags/flag-united-states/

[^1]: The configuration that the user sees is completely independent of the values we are using on the folder and configuration level.

File: translation/pt/patterns/30-day-warranty.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Uma equipe desenvolve um componente que é utilizado em toda a organização. Essa equipe resiste em aceitar ou rejeita completamente contribuições (pedidos de recursos). Esse comportamento bloqueia o progresso e leva a interrupções frequentes devido a escalonamentos.
+
+## Contexto
+
+- As equipes dependem de outra equipe para aceitar suas contribuições para que um componente produzido pela equipe receptora possa ser utilizado pela equipe contribuinte.
+- A equipe receptora não possui os recursos, conhecimento, permissão e/ou inclinação para escrever o componente/pedido de recurso por si própria.
+
+## Forças
+
+- Existe desconfiança em relação às contribuições devido a um histórico de trapaças: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.
```
Comment (DanielleAlmeida) [inline translation/pt/patterns/30-day-warranty.md]: Mudaria de "trapaças" para "incrédulo": "Existe desconfiança em relação às contribuições devido a um histórico incrédulo: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção."

Comment (DanielleAlmeida) [reply translation/pt/patterns/30-day-warranty.md]: I´d like to suggest to change: from the word "cheating" (trapaças" in portguese - Brazil) to "unbelieving" (incrédulos).

Comment (zilio) [reply translation/pt/patterns/30-day-warranty.md]: Olá @DanielleAlmeida, fazendo a adaptação para utilizar o incrédulo, não sei se passa a mensagem adequada para esse tópico, pois "incredulidade" geralmente se refere a uma falta de crença ou dúvida sobre algo, e não necessariamente a atos de enganação ou trapaça.

> Existe desconfiança em relação às contribuições devido a um histórico de incredulidade: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.

Talvez utilizar como os Japoneses fizeram e utilizar a expressão "devido a um histórico de má conduta" para indicar de forma mais clara que no passado os desenvolvedores entregaram código incompleto.

> Existe desconfiança em relação às contribuições devido a um histórico de má conduta: equipes enviaram contribuições incompletas e posteriormente solicitaram correções para torná-las prontas para uso em produção.

Comment (spier) [reply translation/pt/patterns/30-day-warranty.md]: As an aside: Great to see even more collaboration on the translation 🥳 Thank you @DanielleAlmeida!

**Comment by spier** [comment]
@jrcosta one idea that may seem slightly strange but let me share it anyways:

As @zilio and you were already two people working on this translation, one might argue that one round of verification/review of the translation has already happened.

Therefore what we could do is to release the current version of this translation as is, and then let further reviews/fixes of the translations happen one-by-one in further PRs in the future.

It would allow us to get the first version of this translation published quickly, so that people in Brazil can already benefit from it. Even if small mistakes (or wrong translations) may still exist.

What would you think about that approach?

Do we have anything to loose?

**Comment by jrcosta** [comment]
> My opinion on this topic starts with a question (sorry if that has already been answered): Is Brazilian Portuguese so different from European Portuguese that speakers of both languages won't understand written text in the respective other language?

At this juncture, we require a proficient PT-PT speaker to validate. In my opinion, it's easy to use, but I lack extensive knowledge of this language to make this assertion.

> ## My opinion about implementation
> I am leaning for option (2), as it is quick to implement and "correct enough for my point of view.
> 
> Changes required on a folder/configuration level:
> 
> * renaming folder to `pt-br`
> * changing configuration values in the scripts from `pt` to `pt-br`
> 
Taking into account what I mentioned earlier, in my view, option 2 is the most favorable. Can I proceed with implementing the changes to utilize this option, or should I wait for further approval?

**Comment by spier** [comment]
> Can I proceed with implementing the changes to utilize this option, or should I wait for further approval?

@jrcosta yes please, go ahead :)
Doing a `git mv pt pt-br` within the translation folder should get you there.

**Comment by jrcosta** [comment]
> @jrcosta one idea that may seem slightly strange but let me share it anyways:
> 
> As @zilio and you were already two people working on this translation, one might argue that one round of verification/review of the translation has already happened.
> 
> Therefore what we could do is to release the current version of this translation as is, and then let further reviews/fixes of the translations happen one-by-one in further PRs in the future.
> 
> It would allow us to get the first version of this translation published quickly, so that people in Brazil can already benefit from it. Even if small mistakes (or wrong translations) may still exist.
> 
> What would you think about that approach?
> 
> Do we have anything to loose?

Regarding that, following a conversation with @zilio and @daniellealmeida (who were already reviewing it), we share the unanimous opinion that it would indeed be a good idea to publish it now as an initial book. As the community starts using it, corrections may arise, and we will review and implement them.

I would like to request that we also add @daniellealmeida to the list of contributors when the pull request is ready. She was willing and had already begun reviewing. Is this possible, @spier?

**Comment by spier** [comment]
Yes, you can add her to the respective files in the footer. 

We can also add her to the PR as a co-author.

Should I already merge and publish the book, or are there other things that you want to do before that?

**Comment by spier** [comment]
FYI I already put the latest version of this branch on Staging. Looks like it all works fine from the `pt-br` folder:
https://innersourcecommons.gitbook.io/innersource-patterns-staging/v/pt-br/

**Comment by spier** [comment]
@jrcosta I just saw that the pt-br translation of https://innersourcecommons.org was announced today (in slack).

So if you could confirm that I can merge this branch, then I could get this published today as well, making for a nice double punch for Português Brasileiro in the InnerSource Commons :)

**Comment by jrcosta** [comment]
> @jrcosta I just saw that the pt-br translation of https://innersourcecommons.org was announced today (in slack).
> 
> So if you could confirm that I can merge this branch, then I could get this published today as well, making for a nice double punch for Português Brasileiro in the InnerSource Commons :)

Hello, @spier! What fantastic news!

As for the other patterns that need to be translated and the necessary changes, along with mentions and more, I can include them in another pull request. In regard to this one, feel free to publish it.

**Comment by spier** [comment]
@jrcosta @zilio @DanielleAlmeida I can only second what our bot said above: Thank you :)

This has been a fantastic contribution, and super smooth process (from my side at least).

If you want to share any insights into how the contribution process went for you, and maybe what could be improved for contributions of new languages (or translations of new patterns for existing languages) in the future, please do!

Thanks again, you rock!

**Comment by jrcosta** [comment]
> @jrcosta @zilio @DanielleAlmeida I can only second what our bot said above: Thank you :)
> 
> This has been a fantastic contribution, and super smooth process (from my side at least).
> 
> If you want to share any insights into how the contribution process went for you, and maybe what could be improved for contributions of new languages (or translations of new patterns for existing languages) in the future, please do!
> 
> Thanks again, you rock!

I appreciate your words, @spier. As my first contribution to a project, I can say that I felt very at ease, and your support was essential for that. Thank you!

**Pull request merged**